gcc/rust/metadata/rust-export-metadata.cc

```diff
@@ -111,14 +112,12 @@ ExportContext::emit_function (const HIR::Function &fn)
 }
 
 void
-ExportContext::emit_macro (NodeId macro)
+ExportContext::emit_macro (AST::MacroRulesDefinition macro)
```

Comment:
Use a reference here as we don't need to copy the macro: `AST::MacroRulesDefinition &macro`